data/vtlib/target_nchs_alignment.json

```diff
@@ -236,10 +236,10 @@
 		"cmd":[
 			"bam12auxmerge",
 			"level=0",
-			{"subst":"rankstrip_flag","required":"yes","ifnull":{"subst_constructor":{"vals":[ "rankstrip", {"subst":"rankstrip_val", "required":"yes", "ifnull":"1"} ],"postproc":{"op":"concat","pad":"="}}}},
-			"ranksplit=0",
+			{"subst":"rankstrip_nchs_flag","required":"no","ifnull":{"subst_constructor":{"vals":[ "rankstrip", {"subst":"rankstrip_nchs_val", "required":"no", "ifnull":"1"} ],"postproc":{"op":"concat","pad":"="}}}},
```

Comment:
so `rankstrip=1` by default? It that right? (thought we are not adding ranks in dehumanising bowtie2 path?

Reply:
Is it the `"required":"no"` which means the option is just not added now?